apps/api/src/routers/user.py

```diff
@@ -72,6 +72,15 @@ async def apply(
         log.error("User %s has already applied.", user)
         raise HTTPException(status.HTTP_400_BAD_REQUEST)
 
+    raw_app_data_dump = raw_application_data.model_dump()
+
+    for field in ["pronouns", "ethnicity", "school", "major"]:
+        if raw_app_data_dump[field] == "other":
```

Comment:
Thought [non-blocking]: unsure if dump or `__getattr__` would be more performant, but this works fine especially cause we'll need the dump later anyway.